erpnext/patches/v12_0/rename_bank_account_field_in_journal_entry_account.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2019, Frappe and Contributors
+# License: GNU General Public License v3. See license.txt
+
+from __future__ import unicode_literals
+import frappe
+from frappe.utils import getdate, today
+
+def execute():
+	""" Generates leave ledger entries for leave allocation/application/encashment
+		for last allocation """
+	if not frappe.get_meta("Journal Entry Account").has_field("bank_account"):
+		frappe.reload_doc("Accounts", "doctype", "Journal Entry Account")
+		update_journal_entry_account_fieldname()
```

Comment:
Use `rename_field` function to rename the field, it will automatically migrate the data from bank_account_no to bank_account field. It will also take care of other place where the field is used like 'Property Setter', 'report', custom script etc.

Reply:
added changes